specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/monitoring.json

```diff
@@ -180,162 +63,56 @@
             "type": "string"
           },
           {
-            "name": "appId",
-            "description": "The application id.",
-            "in": "path",
-            "required": true,
+            "name": "filter",
+            "in": "query",
+            "required": false,
             "type": "string"
           },
           {
-            "name": "attemptId",
-            "description": "The attempt id.",
-            "in": "path",
-            "required": true,
+            "name": "$orderby",
+            "in": "query",
+            "required": false,
             "type": "string"
           },
           {
-            "$ref": "#/parameters/ApiVersionParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Success",
-            "schema": {
-              "$ref": "#/definitions/HistoryServerDiagnosticResponse"
-            }
-          },
-          "401": {
-            "description": "Unauthorized"
-          }
-        }
-      }
-    },
-    "/monitoring/workloadTypes/spark/pools/{poolName}/livyIds/{livyId}/applications/{appId}/attemptIds/{attemptId}/historyServerGraph": {
-      "get": {
-        "tags": [
-          "Monitoring"
-        ],
-        "operationId": "Monitoring_GetHistoryServerGraph",
-        "description": "Get History Server Graph Data for a given workspace, pool, livyid, appid and attemptId",
-        "x-ms-examples": {
-          "Get history server graph data": {
-            "$ref": "./examples/Monitoring_GetHistoryServerGraph.json"
-          }
-        },
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "name": "poolName",
-            "description": "The spark pool name.",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "livyId",
-            "description": "The livy id.",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "appId",
-            "description": "The application id.",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "attemptId",
-            "description": "The attempt id.",
-            "in": "path",
-            "required": true,
+            "name": "skip",
+            "in": "query",
+            "required": false,
             "type": "string"
-          },
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
           }
         ],
         "responses": {
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/HistoryServerGraphResponse"
+              "$ref": "#/definitions/SqlQueryStringDataModel"
             }
           },
           "401": {
```

Comment:
I know this isn't part of the change, but having a 401 listed here like this indicates that SDKs should treat this as a *success*, is that really what you want?  If this response is intended to generate an error you need to use the [x-ms-error-response](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-error-response) extension.

Reply:
Thank you for your comments. And we think we need not do extra operations about 401, so we decide to remove that from responses.